README.md

```diff
@@ -9,23 +9,28 @@
 
 Federalist is a unified interface for publishing static government websites. It automates common tasks for integrating GitHub and Amazon Web Services to provide a simple way for developers to launch new static websites or more easily manage existing static websites. This repo is home to "federalist-core" - a Node.js app that allows government users to add and configure their Federalist sites.
 
-## Getting started
+## How to set up a local Federalist development server
 
-### Dependencies / Tooling
+### First install these dependencies
 
 Before you start, ensure you have the following installed:
 
 - [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
 - [Docker Compose](https://docs.docker.com/compose/install/#install-compose)
 
-### Running locally (development)
+### Then follow these steps to set up and run your server
+In order to provide a simple development user experience, Federalist has some complexity on the backend. So as part of your local setup, you will need to emulate some of that complexity through the creation steps below. This shouldn't take longer than 15 minutes.
+
+_Note: some terminal commands may take a while to process, without offering feedback to you. Your patience will be rewarded!_
```

Comment:
Which ones? I think all the ones here should print out stuff to the terminal pretty quickly after running.

Reply:
I didn't note which, but a couple of them took a while and didn't give any indication that they were running.